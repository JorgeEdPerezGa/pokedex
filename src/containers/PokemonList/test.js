import React from 'react';
import { shallow } from 'enzyme';
import { PokemonList } from './index';

describe('PokemonList', () => {
  let renderedComponent;

  beforeEach(() => {
    const displayTypesMock = jest.fn();
    renderedComponent = shallow(
      <PokemonList
        pokemonTypes={[]}
      />);
  });

  it('should match snapshot', async () => {
    expect(renderedComponent).toMatchSnapshot();
  });
});
