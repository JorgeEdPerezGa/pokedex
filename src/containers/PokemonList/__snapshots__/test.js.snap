// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PokemonList should match snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <PokemonList
    pokemonTypes={Array []}
  />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <div
        className="loading-pokemon"
      />,
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "className": "loading-pokemon",
      },
      "ref": null,
      "rendered": null,
      "type": "div",
    },
    "type": "section",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <div
          className="loading-pokemon"
        />,
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "className": "loading-pokemon",
        },
        "ref": null,
        "rendered": null,
        "type": "div",
      },
      "type": "section",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
